static/app/components/events/interfaces/spans/spanTreeModel.tsx

```diff
@@ -40,7 +40,9 @@ class SpanTreeModel {
   fetchEmbeddedChildrenState: FetchEmbeddedChildrenState = 'idle';
   showEmbeddedChildren: boolean = false;
   embeddedChildren: Array<SpanTreeModel> = [];
-  showSpanGroup: boolean = false;
+  // This controls if a chain of nested spans that are the only sibling to be visually grouped together or not.
+  // On initial render, they're visually grouped together.
+  showNestedSpanGroup: boolean = false;
```

Comment:
```suggestion
  expandNestedSpanGroup: boolean = false;
```

I found that the wording on this variable was a bit confusing, since it sounds a bit like the whole group itself should be hidden if this is `false`. I'll probably follow up on my changes as well and rename [ungroupedSiblings](https://github.com/getsentry/sentry/blob/feat/autogroup-sibling-spans/static/app/components/events/interfaces/spans/spanTreeModel.tsx#L48) to `expandedSiblingGroups` for the sake of consistency here

Reply:
But now I realize that `expandNestedSpanGroup` sounds very similar to `toggleNestedSpanGroup`... `isNestedSpanGroupExpanded` makes it a lot more clear, but it's quite a long name

Reply:
@0Calories `isNestedSpanGroupExpanded` works for me. I've renamed `expandNestedSpanGroup` to `isNestedSpanGroupExpanded`.